docs/core/tools/csproj.md

```diff
@@ -67,8 +67,17 @@ Setting this property to `false` will override implicit inclusion and the behavi
 This change does not modify the main mechanics of other includes. However, if you wish to specify, for example, some files to get published with your app, you can still use the known mechanisms in *csproj* for that (for example, the `<Content>` element).
 
 ### Recommendation
-With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
+With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (for example, runtime and NuGet packaging).
 
+## How to see the whole project as MSBuild sees it
+
+While those csproj changes greatly simplify project files, you might want to see the fully expanded project as MSBuild sees it once the SDK and its targets are included. Preprocess the project with the `/pp` switch of the [`dotnet msbuild`](dotnet-msbuild.md) command, which shows which files are imported, their sources, and their contributions to the build without actually building the project:
```

Comment:
That actually brings up a point maybe worth mentioning here: you can combine `/pp:out` with a `path/to/project` to preprocess _that project_. Or just do it in a directory to get the project that MSBuild would build in that directory.

So both of these are valid:
```
msbuild /pp:preprocessed.xml c:\src\Path\to\library.csproj
msbuild /pp:preprocessed.xml
```

Where the latter will preprocess the project in the current working directory, or error if there's more than one.

That's the same way that all MSBuild invocations work so it probably doesn't need to be mentioned here. But while we're talking about it . . .